aspnetcore/fundamentals/logging/index.md

```diff
@@ -388,8 +388,10 @@ ASP.NET Core defines the following log levels, ordered here from lowest to highe
 
 Use the log level to control how much log output is written to a particular storage medium or display window. For example:
 
-* In production, send `Trace` through `Information` level to a volume data store. Send `Warning` through `Critical` to a value data store.
-* During development, send `Warning` through `Critical` to the console, and add `Trace` through `Information` when troubleshooting.
+* In production, logging at the `Trace` through `Information` levels produces a high-volume of detailed log messages. To control costs and not exceed data storage limits, log `Trace` through `Information` level messages to a high-volume, low-cost data store. Because logging at `Warning` through `Critical` levels typically produces fewer, smaller log messages, any data store choice is usually acceptable.
```

Comment:
* In production, logging at the `Trace` through `Information` levels can produce a high-volume of detailed log messages. To limit data storage costs, log `Trace` through `Information` level messages to a high-volume, low-cost data store. Logging of `Warning` through `Critical` levels typically produces fewer log messages, therefore the data store choice is more flexible.

Reply:
16 words
21 words
20 words

Having a cutoff of 25 words isn't really a cutoff *if 25 words isn't the cutoff*. :smile:

Reply:
🤔 I'm not really feel'in the first suggestion.

> To control costs and not exceed data storage limits, log `Trace` through `Information` level messages to a high-volume, low-cost data store.

... to ...

> To limit data storage costs, log Trace through Information level messages to a high-volume, low-cost data store.

... fails to relay the concept that data storage limits are often a problem in high-volume logging scenarios.

> Because logging at `Warning` through `Critical` levels typically produces fewer, smaller log messages, any data store choice is usually acceptable.

... to ...

> Logging of Warning through Critical levels typically produces fewer log messages, therefore the data store choice is more flexible.

... doesn't matter too much to me, except that I'm not a fan of the conjunctive adverb over the prepositional phrase. Your suggestion would require a semi-colon-comma structure ("; therefore,") or splitting the sentence. However, I'm not a huge fan of my list comma and phrase comma. I'll go with the "therefore," but I'll use a 2nd sentence for that one.